frontend/package.json

```diff
@@ -12,8 +12,8 @@
     "buildAppFast": "yarn workspace @streamlit/app buildFast",
     "typecheck": "yarn workspace @streamlit/lib typecheck && yarn workspace @streamlit/app typecheck",
     "lint": "yarn lintApp && yarn lintLib",
-    "lintApp": "yarn workspace @streamlit/app lint",
-    "lintLib": "yarn workspace @streamlit/lib lint",
```

Comment:
Was `yarn workspace @streamlit/lib lint` not working correctly? Or why did you moved the full command to this package.json?

Reply:
it was working but because the `eslintrc.js` was not working with relative paths, I just changed it to be the root package.json to run the commands. However, I think technically, this could be changed back since I changed our `eslintrc.js`

Reply:
yeah, I think it would be slightly better to change this back if possible. Than you are also able to run `yarn lint` directly from `lib` or `app` folder

Reply:
Sounds good. I'll just directly commit to the `feature/st-lib` branch if that's ok. Otherwise lmk and I can make a separate PR.